lib/controller_patches.rb

```diff
@@ -7,14 +7,8 @@
 Rails.configuration.to_prepare do
   # Front page needs some additional info
   GeneralController.class_eval do
-      # Make sure it doesn't break if blog is not available
-      def frontpage
-          begin
-              blog
-          rescue
-              @blog_items = []
-              @twitter_user = MySociety::Config.get('TWITTER_USERNAME', '')
-          end
-      end
+    def frontpage
+      blog
```

Comment:
The [core controller blog method](https://github.com/mysociety/alaveteli/blob/0.24.1.7/app/controllers/general_controller.rb#L30-L56) handles the feed timeout and sets `@blog_items` for us